packages/application-components/src/components/modal-page/internals/modal-page.styles.js

```diff
@@ -0,0 +1,52 @@
+import { css, styled, keyframes } from '@emotion/core';
```

Comment:
does this work? I thought there was no styled export in `@emotion/core` 🤔 

[ref](https://github.com/emotion-js/emotion/blob/master/packages/core/src/index.js)

the docs say to do `import styled from '@emotion/styled` [docs](https://emotion.sh/docs/styled)

Reply:
I haven't tested it yet (hence the WIP state). Thanks for pointing this out 😄